### PR TITLE
Using simpler wrapper function capabilities.Setup instead of calling Initialize directly

### PR DESCRIPTION
**Pull request by SataQiu**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Using simpler wrapper function `capabilities.Setup` instead of calling `Initialize` directly

#### Which issue(s) this PR fixes:

Ref #77820

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by SataQiu** [comment]
/assign @liggitt 
PTAL

File: cmd/kube-apiserver/app/server.go [hunk 1]
```diff
@@ -250,16 +250,7 @@ func CreateKubeAPIServerConfig(s completedServerRunOptions) (
...
-		AllowPrivileged: s.AllowPrivileged,
-		// TODO(vmarmol): Implement support for HostNetworkSources.
-		PrivilegedSources: capabilities.PrivilegedSources{
-			HostNetworkSources: []string{},
-			HostPIDSources:     []string{},
-			HostIPCSources:     []string{},
-		},
-		PerConnectionBandwidthLimitBytesPerSec: s.MaxConnectionBytesPerSec,
-	})
+	capabilities.Setup(s.AllowPrivileged, s.MaxConnectionBytesPerSec)
```
Comment (liggitt) [inline cmd/kube-apiserver/app/server.go:253]: this is not equivalent... specifically, the Setup call is not guarded by the once.Do check

Comment (liggitt) [reply cmd/kube-apiserver/app/server.go:253]: sigh... sorry... read too quickly, Setup delegates to Initialize 🤦‍♂️

**Comment by liggitt** [comment]
unit test failure is a known test-infra config issue being worked on in https://github.com/kubernetes/test-infra/pull/27515#issuecomment-1248245618

**Pull request merged**